br/pkg/task/backup_raw.go

```diff
@@ -50,6 +50,11 @@ func DefineRawBackupFlags(command *cobra.Command) {
 
 	// This flag can impact the online cluster, so hide it in case of abuse.
 	_ = command.Flags().MarkHidden(flagRemoveSchedulers)
+
+	_ = command.Flags().MarkHidden(flagRemoveSchedulers)
```

Comment:
Duplicated with above line.

Reply:
Consider also hide `flagCompressionType`. Just one default compress type is enough.

Reply:
ok, make sense.